webapi/src/main/scala/org/knora/webapi/messages/admin/responder/projectsmessages/ProjectsMessagesADM.scala

```diff
@@ -511,6 +511,11 @@ sealed trait ProjectIdentifierADM { self =>
       case ShortnameIdentifier(value) => Some(value.value)
       case _                          => None
     }
+  def valueAsString: String = self match {
```

Comment:
What is the point of adding this method?

The exact same implementation is already present as `org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM#getId`

Reply:
true, overlooked this one. Will remove it